helpers/manifest-parser.js

```diff
@@ -139,8 +139,7 @@ function parseIcon(raw) {
   let sizes = parseString(raw.sizes);
   if (sizes.value !== undefined) {
     let set = new Set();
-    sizes.value.split(/\s/).forEach(size => set.add(size.toLowerCase()));
-
+    sizes.value.trim().split(/\s+/).forEach(size => set.add(size.toLowerCase()));
```

Comment:
should it be trimmed, though? That's not clear to me from the spec. If so, you should actually set `let sizes = parseString(raw.sizes, true);` up above to match the other cases where strings are trimmed, then have `sizes.value.split(/\s+/).forEach(size => set.add(size.toLowerCase()));` here